l1-contracts/contracts/state-transition/chain-deps/ZKChainStorage.sol

```diff
@@ -169,4 +169,10 @@ struct ZKChainStorage {
     address settlementLayer;
     /// @dev Priority tree, the new data structure for priority queue
     PriorityTree.Tree priorityTree;
+    /// @dev The address of the PLONK Verifier contract
+    address plonkVerifier;
+    /// @dev The address of the FFLONK Verifier contract
+    address fflonkVerifier;
+    /// @dev The length of the FFLONK proof type
+    uint256 fflonkProofLength;
 }
```

Comment:
We don't need it here. This is a storage for ZKChain contract, not for verifier. Verifier can contain it itself

Reply:
Was planning to keep the DualVerifier modular with ZKChainStorage, where if we want to simply upgrade a PLONK/FFLONK verifier, it could be easily possible. I might be wrong here though.